test/core/utils/TransactionMapping.spec.ts

```diff
@@ -1616,4 +1623,45 @@ describe('TransactionMapping - createFromDTO (Transaction.toJSON() feed)', () =>
                 'CBD67E2DBA5D69157CE32874EFDD680E41B1BFFD12B781F84E8AD883920BBB50EA38AFE078E99EE5EE4BDFDA77E8C101EBD3100C0D471673471A61B23E513FC7E21F7803316B906A688F14AA75002913A3B57DD13469BC27CF8C82FD5C4C76867011AEDC7C4870D8C5AF9C175F0DA5A8E2AD3A327D868BFBA34A5E3D',
         );
     });
+
+    it('should serialize, deserialize top level MosaicSupplyRevocationTransaction', () => {
+        const mosaicId = new MosaicId('0DC67FBE1CAD29E5');
+        const transaction = MosaicSupplyRevocationTransaction.create(
+            Deadline.createFromDTO('100'),
+            account.address,
+            new Mosaic(mosaicId, UInt64.fromUint(5)),
+            NetworkType.TEST_NET,
+        );
+
+        const expectedPayload =
+            'A8000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000001984D43000000000000000064000000000000009826D27E1D0A26CA4E316F901E23E55C8711DB20DFD26776E529AD1CBE7FC60D0500000000000000';
+        expect(transaction.serialize()).eq(expectedPayload);
+
+        const transactionFromPayload = TransactionMapping.createFromPayload(expectedPayload);
+        expect(transactionFromPayload).deep.equal(transaction);
+    });
+
+    it('should serialize, deserialize aggregate MosaicSupplyRevocationTransaction', () => {
```

Comment:
why do you want a separate aggregate test, it doesn't seem like (m)any others have a dedicated aggregate test?

Reply:
Moved to a common aggregate test example